src/main/java/org/openapitools/configuration/OpenAPIDocumentationConfig.java

```diff
@@ -29,7 +29,7 @@ ApiInfo apiInfo() {
             .license("")
             .licenseUrl("http://unlicense.org")
             .termsOfServiceUrl("")
-            .version("1.0")
+            .version("1.0.0")
```

Comment:
In case you change anything in the openapi.yaml you should increment one of the version numbers to be able to version the changes. This is especially important for the front-end.

Reply:
Ok.. I will note that and take care in the future commits